applications/serial_lte_modem/src/slm_at_udp_proxy.c

```diff
@@ -516,6 +564,9 @@ int handle_at_udp_client(enum at_cmd_type cmd_type)
 			proxy.sec_tag  = INVALID_SEC_TAG;
 			if (at_params_valid_count_get(&at_param_list) > 4) {
 				at_params_unsigned_int_get(&at_param_list, 4, &proxy.sec_tag);
+				if (at_params_valid_count_get(&at_param_list) > 5) {
+					at_params_int_get(&at_param_list, 5, &proxy.conn_id);
```

Comment:
We should verify here that `conn_id` is either 0, 1 or 2, so we can feed it directly to Socket API.

Reply:
Yes we can do that